metafeed-finder.js

```diff
@@ -40,23 +59,25 @@ module.exports = class MetafeedFinder {
   }
 
   fetch(mainFeedId, cb) {
-    if (this._map.has(mainFeedId)) {
-      const metaFeedId = this._map.get(mainFeedId)
-      cb(null, metaFeedId)
-    } else if (mainFeedId === this._ssb.id) {
-      this._ssb.metafeeds.findOrCreate((err, rootMF) => {
-        if (err) cb(err)
-        else if (!rootMF) cb(null, null)
-        else {
-          const metaFeedId = rootMF.id
-          this._map.set(mainFeedId, metaFeedId)
-          this._inverseMap.set(metaFeedId, mainFeedId)
-          cb(null, metaFeedId)
-        }
-      })
-    } else {
-      this._request(mainFeedId, cb)
-    }
+    this._onLoaded.onReady(() => {
+      if (this._map.has(mainFeedId)) {
+        const metaFeedId = this._map.get(mainFeedId)
+        cb(null, metaFeedId)
+      } else if (mainFeedId === this._ssb.id) {
+        this._ssb.metafeeds.findOrCreate((err, rootMF) => {
+          if (err) cb(err)
+          else if (!rootMF) cb(null, null)
+          else {
+            const metaFeedId = rootMF.id
+            this._map.set(mainFeedId, metaFeedId)
+            this._inverseMap.set(metaFeedId, mainFeedId)
+            cb(null, metaFeedId)
+          }
+        })
+      } else {
+        this._request(mainFeedId, cb)
+      }
+    })
```

Comment:
Everything inside the `this._onLoaded.onReady(() => {` is just indented, but the diff makes it hard to see.